cmd/util/publisher/publisher.go

```diff
@@ -77,14 +78,15 @@ func (options *PublisherOptions) run(cmd *cobra.Command, args []string) {
 
 	started := options.StartWith == ""
 
-	keys := make([]string, 0, len(camel.Runtime.Artifacts))
-	for k := range camel.Runtime.Artifacts {
+	catalog := camel.Runtime(defaults.CamelVersion)
```

Comment:
Shouldn't be `p.Spec.Build.CamelVersion`? I imagine it's the same physically, still may be better logically...